Source/town.h

```diff
@@ -1,17 +1,30 @@
 /**
- * @file town.h
- *
- * Interface of functionality for rendering the town, towners and calling other render routines.
- */
+* @file town.h
+*
+* Interface of functionality for rendering the town, towners and calling other render routines.
+*/
+
 #pragma once
 
 #include "gendung.h"
 #include "interfac.h"
 
 namespace devilution {
 
+/**
+* @brief Update the map to show the open hive
+*/
 void TownOpenHive();
+
+/** 
+* @brief Update the map to show the open grave
+*/

```

Comment:
Doxygen comments should have the stars aligned:
```suggestion
/** 
 * @brief Update the map to show the open grave
 */
```

Reply:
ok